.github/workflows/jit.yml

```diff
@@ -32,9 +32,27 @@ jobs:
     timeout-minutes: 90
     steps:
       - uses: actions/checkout@v4
-      - name: Build tier two interpreter
+      - name: Configure tier two interpreter
         run: |
           ./configure --enable-experimental-jit=interpreter --with-pydebug
+      - name: Regen JIT
+        run: |
+          make regen-jit
+      - name: Check for changes
+        run: |
+          git add -u
+          changes=$(git status --porcelain)
+          # Check for changes in regenerated files
+          if test -n "$changes"; then
+            echo "Generated files not up to date."
+            echo "Perhaps you forgot to run make regen-jit. ;)"
+            echo "$changes"
+            echo ""
+            git diff --staged || true
+            exit 1
+          fi
+      - name: Build tier two interpreter
+        run: |
```

Comment:
Can you revert the changes in this file? `jit_stencils.h` isn't checked in, and it's built automatically by the default `all` target. So there's no need to run `make regen-jit` manually or check whether it's "up-to-date".